static/library/line_chart_component.ts

```diff
@@ -110,6 +116,7 @@ export class DatacommonsLineComponent extends LitElement {
       })),
       svgChartHeight: 200,
       title: this.header || this.title,
+      statVarNameRegex: this.variableNameRegex
```

Comment:
super nit: use consistent name for it

Reply:
right now users specify "variables" in the web component which in turn maps to "statVarSpecs" used by the line tile. I tried to keep the name consistent with how each component references variable/statVar .. so not sure if we should consistently use statVarNameRegex or variableNameRegex here

Reply:
i see, ok, that makes sense!

Reply:
I like `variableNameRegex` for the web component attribute to be consistent with the `variable` web component attribute